app/src/views/examples/exit-this-page-with-skiplink/index.njk

```diff
@@ -0,0 +1,20 @@
+{% extends "layout.njk" %}
```

Comment:
Thanks for adding this and the full page example! Would also be good to add one to the 'Translated' example that shows the component in its translated state.